app/templates/calendar_day_view.html

```diff
@@ -1,95 +1,78 @@
 {% extends "partials/calendar/calendar_base.html" %}
 {% block body %}
 <div id="day-view">
-    <link rel="stylesheet" type="text/css"
-          href="{{ url_for('static', path='/dayview.css') }}">
-    <div id="top-tab"
-         class="d-flex justify-content-around align-items-center sticky-top">
-        {% if view == 'day' %}
-        <a href="/closethedayview" title="close day view"><img class="pb-1"
-                                                               src="{{ url_for('static', path='/images/icons/close_sidebar.svg')}}"
-                                                               width="15em"
-                                                               height="15em"></a>
-        <span id="month-num"
-              class="month-title fw-bold text-white">{{month}}</span>
-        <span id="day-num" class="day-title fw-bold text-white">{{day}}</span>
-        {% if zodiac %}
-        <div class="zodiac-sign">
-            <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}"
-               title="Zodiac-sign: {{zodiac.name}}"
-               target="_blank"><img
-                    src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}"
-                    alt="zodiac sign" width="14em" height="13em"></a>
-        </div>
-        {% endif %}
-        {% else %}
-        <span class="fw-bold text-white date-nums">{{day}} / {{month}}</span>
-        {% endif %}
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', path='/dayview.css') }}">
+  <div id="top-tab" class="d-flex justify-content-around align-items-center sticky-top">
+    {% if view == 'day' %}
+    <a href="/closethedayview" title="close day view"><img class="pb-1"
+                                                           src="{{ url_for('static', path='/images/icons/close_sidebar.svg')}}"
+                                                           width="15em" height="15em"></a>
+    <span id="month-num" class="month-title fw-bold text-white">{{month}}</span>
+    <span id="day-num" class="day-title fw-bold text-white">{{day}}</span>
+    {% if zodiac %}
+    <div class="zodiac-sign">
+      <a href="https://www.horoscope.com/zodiac-signs/{{zodiac.name}}" title="Zodiac-sign: {{zodiac.name}}"
+         target="_blank"><img src="{{ url_for('static', path='/images/zodiac/' + zodiac.name + '.svg') }}"
+                              alt="zodiac sign" width="14em" height="13em"></a>
     </div>
-    {% if international_day %}
-    <div class="international-days">
-        The International days are: "{{ international_day.international_day }}"
+    {% endif %}
+    {% if parasha %}
+    <div class="parashot">
+        {{ gettext(" :פרשת השבוע %(hebrew)s", hebrew=parasha.name) }}
+        <br>{{ gettext("%(name)s", name=parasha.hebrew_name) }}
+        <a href={{parasha.link}} target="_blank">{{ gettext(" פירוט בלינק נפרד על %(parashah)s", parashah=parasha.hebrew_name) }}</a>
```

Comment:
Change to English (the default in our project)

Reply:
i edit